### DISCUSSION TITLE
Publish NPM package which only contains CSS

### DISCUSSION THREAD

**Discussion by JinIgarashi** [undp/design-system#1278] [Q&A] [answered]
We are implementing UNDP design component in sveltekit in the below repo.

https://github.com/UNDP-Data/geohub/tree/develop/packages/svelte-undp-design

There is a `@undp/design-system` package in NPM, but the size of package is too big, I don't want to use this package in our repo. It is difficult for us to track your changes on CSS.

https://www.npmjs.com/package/@undp/design-system?activeTab=code

Could you kindly consider publishing new package which only contain CSS? So, we can import UNDP design CSS to implement svelte component.

**Comment by Jura** [comment, marked as answer]
@JinIgarashi , if you need just to include css in your project you can do it via jsdelivr CDN: https://cdn.jsdelivr.net/npm/@undp/design-system/ . Use instructions for specific components at https://design.undp.org/ to identify specific files to include.

If you need to work with SCSS files importing them and amending - you will have to keep this package as a dependency and import respective assets into yours

**Comment by JinIgarashi** [comment]
As I mentioned, this repository's NPM package size is too big because it contains all source code. You should use `files` property in package.json to include only files after building. Otherwise, no one can use this repository as dependency.

**Comment by Jura** [comment]
On the contrary, anybody can use repository as a dependency as it doesn't need to be included in your package, you can import what you need only, disregarding everything else. We have several projects, starting with starter's template, which are successfully using design system package as dependency. It is really up to you how to manage imports/inclusions in your package

**Comment by JinIgarashi** [comment]
well, the NPM package can be used if you want to download 64MB everytime...

**Comment by Jura** [comment]
@JinIgarashi , if you are still interested - we've just published a separate NPM package [@undp/design-system-assets](https://www.npmjs.com/package/@undp/design-system-assets), containing only compiled assets (css/js/fonts/images). 
`npm i @undp/design-system-assets`
